File: Java/problem001.java
/* * Solution to Project Euler problem 1 in Java */

package project.euler;

public class problem001
{
    public static void main(String[] args)
    {
        int total = 0;
		for(int i = 1; i < 1000; i ++)
		{
			if(i % 3 == 0 || i % 5 == 0)
                        {   total = total + i;
                        }
		}
		System.out.println(total);
    }
    
}

File: C-Cpp/problem001.cpp
/* * Solution to Project Euler problem 1 in C++ */
#include <cstdlib> 
#include "iostream"  // less than and greater than are disabled

using namespace std; 
int main(int argc, char** argv)
{
  int sum =0;  
  for(int i = 0; i < 1000; i++)
    if( (i%3 == 0) || (i % 5 == 0) )
    sum = sum + i;
    
  cout << sum << endl;
  return 0;
}
